manticore/platforms/linux.py

```diff
@@ -2395,9 +2395,13 @@ def sys_open(self, buf, flags, mode):
 
     def generate_workspace_files(self):
         def solve_to_fd(data, fd):
```

Comment:
Wishful thinking: since this is a public function it would be nice to have a `"""..."""` docstring here.